ext/groonga/extconf.rb

```diff
@@ -230,21 +230,27 @@ def build_groonga_from_git(major, minor, micro)
   message(" done\n")
 end
 
-def build_groonga_from_tar_gz(major, minor, micro)
-  tar_gz = "groonga-#{major}.#{minor}.#{micro}.tar.gz"
+def build_groonga_from_tar_gz
+  tar_gz = "groonga-latest.tar.gz"
   url = "http://packages.groonga.org/source/groonga/#{tar_gz}"
+  groonga_source_dir = "./groonga-latest"
 
   download(url)
 
+  if Dir.exist?(groonga_source_dir)
+    FileUtils.rm_rf(groonga_source_dir)
```

Comment:
In this clause, we will get `no such directory error` because we don't create a new empty directory.